src/action/ComputeCurvatureAction.cpp

```diff
@@ -33,7 +33,7 @@ void ComputeCurvatureAction::performAction(
 
   if (getMesh()->getDimensions() == 2) {
     dataValues = Eigen::VectorXd::Zero(dataValues.size());
-    //assign(dataValues) = 0.0;
+    //assign(values) = 0.0;
```

Comment:
:thinking: Why that?

Reply:
I think this got automatically renamed, but we can also just kill the comment, if you want? To me it does not really look like important informaiton.

Reply:
Good practice would be to never leave commented out code. But we are not living in a perfect world :see_no_evil:

Reply:
It's gone now 👍 One step closer to a perfect world 🚀